services/wallet/datastore.go

```diff
@@ -973,7 +973,7 @@ func (pg *Postgres) ConnectCustodialWallet(ctx context.Context, cl *CustodianLin
 		) values (
 			$1, $2, $3
 		)
-		on conflict (wallet_id, custodian, linking_id) 
```

Comment:
My editor is set up to remove trailing spaces, so here and below in this file it did just that.